app/views/finance/ecf/statements/show.html.erb

```diff
@@ -17,101 +17,11 @@
-          </p>
-
-          <p class="govuk-body-s govuk-!-margin-bottom-2">
-            Additional adjustments <span><%= number_to_pounds(@calculator.additional_adjustments_total) %></span>
-          </p>
-
-          <p class="govuk-body-s govuk-!-margin-bottom-2">
-            VAT <span><%= number_to_pounds(@calculator.vat) %></span>
-          </p>
-        </div>
-      </div>
-
-      <div class="finance-panel__summary__meta">
-        <div class="finance-panel__summary__meta__dates">
-          <div>
-            <div>
-              <strong class="govuk-body-s govuk-!-margin-bottom-0">Milestone cut off date</strong>
-            </div>
-            <div class="govuk-heading-m govuk-!-padding-top-0">
-              <%= @statement.deadline_date.to_fs(:govuk) %>
-            </div>
-          </div>
-
-          <div>
-            <div>
-              <strong class="govuk-body-s govuk-!-margin-bottom-0">Payment date</strong>
-            </div>
-            <div class="govuk-heading-m govuk-!-padding-top-0">
-              <%= @statement.payment_date.to_fs(:govuk) %>
-            </div>
-          </div>
-        </div>
-
-        <div class="finance-panel__summary__meta__counts">
-          <div class="second govuk-list govuk-!-margin-bottom-0">
-            <strong class="govuk-body-s govuk-!-margin-bottom-0">Total starts</strong>
-            <div class="govuk-heading-m govuk-!-padding-top-0">
-              <%= @calculator.started_count %>
-            </div>
-          </div>
-
-          <div>
-            <strong class="govuk-body-s govuk-!-margin-bottom-0">Total retained</strong>
-            <div class="govuk-heading-m govuk-!-padding-top-0">
-              <%= @calculator.retained_count %>
-            </div>
-          </div>
-
-          <div>
-            <strong class="govuk-body-s govuk-!-margin-bottom-0">Total completed</strong>
-            <div class="govuk-heading-m govuk-!-padding-top-0">
-              <%= @calculator.completed_count %>
-            </div>
-          </div>
-
-          <% if @calculator.extended_count.positive? %>
-            <div>
-              <strong class="govuk-body-s govuk-!-margin-bottom-0">Total extended</strong>
-              <div class="govuk-heading-m govuk-!-padding-top-0">
-                <%= @calculator.extended_count %>
-              </div>
-            </div>
-          <% end %>
-
-          <div>
-            <strong class="govuk-body-s govuk-!-margin-bottom-0">Total voided</strong>
-
-            <div class="govuk-heading-m govuk-!-padding-top-0 govuk-!-margin-bottom-0">
-              <%= @calculator.voided_count %>
-            </div>
-
-            <%= govuk_link_to 'View<span class="govuk-visually-hidden"> voided declarations</span>'.html_safe, finance_ecf_payment_breakdown_statement_voided_path(@ecf_lead_provider.id, @statement), class: "govuk-body-s" %>
-          </div>
-        </div>
-      </div>
-    </div>
+    <% if @statement.cohort.mentor_funding? %>
```

Comment:
I wonder if we can make the decision once and show different partials than have this check multiple times in the show view?